src/System.Management.Automation/engine/NativeCommandParameterBinder.cs

```diff
@@ -426,8 +434,16 @@ private void PossiblyGlobArg(string arg, CommandParameterInternal parameter, boo
 
             if (!argExpanded)
             {
-                _arguments.Append(arg);
-                AddToArgumentList(parameter, arg);
+                if (ExperimentalFeature.IsEnabled(ExperimentalFeature.PSNativeScriptBlockArgumentFeatureName) && parameter.ArgumentValue is ScriptBlock sb)
+                {
+                    _arguments.Append(sb.Ast.Extent.Text);
```

Comment:
Can you add a comment here that because the extent text is added as-is as a single argument, this behaves as though the script block was quoted text rather than whitespace within the scriptblock being passed as individual arguments to the native command

Reply:
sure